### PR TITLE
Windows/NPM Update release.json for 1.3.2 driver

### PR DESCRIPTION
**Pull request by derekwbrown**
### What does this PR do?

- Includes 1.3.2 driver (signed) 
### Motivation

Includes new NPM driver with additional shutdown BSOD fix.

### Reviewer's Checklist
- [x] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [x] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.

### CODE REVIEW

File: releasenotes/notes/npm132driver-f59bb5ae5effecce.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    On Windows, includes NPM driver 1.3.2 which has a fix for a BSOD on system probe shutdown.
```
Comment (maycmlee) [inline releasenotes/notes/npm132driver-f59bb5ae5effecce.yaml:11]:
```suggestion
    For Windows, includes NPM driver 1.3.2, which has a fix for a BSOD on system probe shutdown.
```

**Review by maycmlee — APPROVED** [review]
Small nit

File: pkg/network/tracer/tracer_test.go [hunk 2]
```diff
@@ -1636,3 +1636,9 @@ func skipIfWindows(t *testing.T) {
 		t.Skip("test unavailable on windows")
 	}
 }
+
+func skipIfNotWindows(t *testing.T) {
+	if runtime.GOOS != "windows" {
+		t.Skip("test only available on windows")
+	}
+}
```
Comment (djmitche) [inline pkg/network/tracer/tracer_test.go:1644]: Is this included accidentally?

Comment (derekwbrown) [reply pkg/network/tracer/tracer_test.go:1644]: yeah.  I originally had a test (that was windows only), but the test didn't actually work so I backed it out.
I'll remove when I update to the signed driver.

**Review by brycekahle — APPROVED** [review]

**Review by KSerrania — COMMENTED** [review]
If this is to be included in the next release, could you also update the `release-a6` and `release-a7` sections?

**Review by KSerrania — APPROVED** [review]

**Pull request merged**